streaming/python/ray/streaming/runtime/queue/queue_interface.py

```diff
@@ -0,0 +1,205 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from abc import ABCMeta, abstractmethod
+from enum import Enum
+
+import ray
+import ray.streaming.runtime.queue.queue_utils as queue_utils
+
+
+class QueueConfig:
+    """
+    queue config
+    """
+    # operator type
+    OPERATOR_TYPE = "operator_type"
+
+    # reliability level
+    RELIABILITY_LEVEL = "reliability_level"
+
+
+class OperatorType(Enum):
+    """
+    operator type
+    """
+    SOURCE = 1
+    TRANSFORM = 2
+    SINK = 3
+
+
+class ReliabilityLevel(Enum):
+    """
+    reliability level
```

Comment:
Please improve the documentation for any public APIs.

Reply:
removed